config/locales/es.yml

```diff
@@ -1819,8 +1819,9 @@ es:
           title: Edita la información básica de %{name}
         dependents:
           no_eligible_dependents:
-            subtitle: No recibirá el Crédito Tributario por Hijos porque no tiene dependientes elegibles.
-            title: 'Usted ha eliminado a todos sus dependientes elegibles para CTC. '
+            credits: CTC y EITC
+            subtitle: No recibirá %{ineligible_credits} porque no tiene dependientes elegibles.
```

Comment:
Does this string interpolation make sense in Spanish? I'm concerned it doesn't, and am unsure.

Reply:
Nice `credits` thing